packages/components/datepicker/package.json

```diff
@@ -16,7 +16,7 @@
     "@contentful/f36-tokens": "^4.0.0",
     "emotion": "^10.0.17",
     "date-fns": "^2.28.0",
-    "react-day-picker": "^8.0.7",
+    "react-day-picker": "8.2.1",
```

Comment:
Do we want to ignore the patches as well? I'm not sure about the pinning part

Reply:
I don't think we should pin.

Reply:
classnames should not change, but there was already a case when they introduced a fix and therefore changed a pseudo-class from aria-disabled to disabled. What problem do you see with pinning this dependency?

Reply:
I thought dependabot would still create for us PRs to bump this package, no?

Reply:
dependabot will stop bumping versions, i think we can cover such cases with tests

Reply:
How would you cover it with tests?
And the additional problem is that you can get a newer version of `react-day-picker` in exp-packages for example and you will get broken styles there.

Reply:
if I allow patch updates, will dependabot work?

Reply:
> How would you cover it with tests?

Basically check if the classnames you're overriding being rendered when the component mount, it should be fairly simple.

Is experience-packages consuming react-day-picker directly? Isn't it consuming datepicker from f36? So broken styles will never reach the packages if tests are failing in dependabot PRs

Reply:
Btw, why do you think dependabot will stop bumping this package? 
It says that by default it will bump every dep defined in the package.json
<img width="756" alt="image" src="https://user-images.githubusercontent.com/10744462/189688940-19676067-8715-4ef3-8042-0b1528385f1d.png">

Reply:
>  And the additional problem is that you can get a newer version of react-day-picker in exp-packages for example and you will get broken styles there. It is actually what happened to me right now.

If styles can break like that we should probably pin. But it seems brittle, maybe we can make the implementation more robust later on.

Reply:
Also chromatic visual diffs should break when styles change 🤔

Reply:
> Also chromatic visual diffs should break when styles change 🤔

But that's the problem if we don't pin, consumers can have a different version of react-day-picker than the one we've built with.

Reply:
I'd say if the pinned dep is such a problem then we can unpin it back. Because I've already changed all the pseudo-classes to regular ones, except the `dropdown`. The regular class names should not change easily, it's gonna be major update and we should catch it with our styles types.